### ISSUE TITLE
release the source of the latest version

### ISSUE THREAD

**Issue by blshkv** [languagetool-org/languagetool-browser-addon#241]
Subj. The latest available version via 
https://chrome.google.com/webstore/detail/grammar-and-spell-checker/oldceeleldhonbafppcapldpdifcinji?hl=en is dated as Apr 2019, however it were no changes at github since 2018. Please release the code.

**Comment by Mchar7** [comment]
Indeed. I just found out about about the LanguageTool add-on, yet I do not feel comfortable using it if you are no longer providing the source. It's sad considering the LanguageTool add-on is a really nice piece of software.

**Comment by fluks** [comment]
If you don't want to license the whole product as open source, could you perhaps release the highlighting code as an open source library? I'm making a Finnish spellchecking browser extension and highlighting textareas is so tricky. I'm using an open source library for it now, but it doesn't work always.

**Comment by felschr** [comment]
It's somewhat ironic that on the settings page of the extension it states "© 2020 LanguageTooler GmbH – We ❤ Open Source".
